README.md

```diff
@@ -57,9 +60,33 @@ unless real values don't help users know what to change.
 
 ```hcl
 
+module "watsonx_assistant" {
+  source            = "terraform-ibm-modules/watsonx-assistant/ibm"
+  region            = "us-south"
+  assistant_name    = "example-wx-assistant-basic"
+  plan              = "lite"
+  resource_group_id = module.resource_group.resource_group_id
```

Comment:
dont use `module...` in the usage. Do something like this: 
`resource_group_id = "a8c...8230a" # replace with ID of resource group`